drivers/github/index.js

```diff
@@ -13,11 +13,15 @@ module.exports =  {
 
 function createRepo(options) {
     let githubConfig = _.find(config.get().drivers, { type: 'github' });
-    const url = `https://api.github.com/user/repos?access_token=${githubConfig.accessToken}`;
+    const url = `https://api.github.com/user/repos`;
     return axios.post(url, {
         name: options.name,
         description: "created by bitcar",
         private: options.private || false
+    }, {
+        headers: {
```

Comment:
Have you been able to test what happens if there isn't an access token?

Reply:
I forgot about that, will do.

Reply:
done.